### PR TITLE
v8.0 what's new

### PR DESCRIPTION
**Pull request by Pessimistress**
For #3983

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -4,7 +4,67 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
...
+- `ScreenGridlayer` (GPU aggregation only)
+- `ContourLayer` (GPU aggregation only)
+- `GridLayer` (GPU aggregation only)
+
+### Other New Features and Improvements
+
+- Render to a frame buffer by specifying the `_framebuffer` prop of [Deck](/docs/api-reference/deck.md).
+- Pick a 3d surface point in the scene by passing `unproject3D: true` to `deck.pickObject` or `deck.pickMultipleObjects`.
+- [ArcLayer](/docs/layers/arc-layer.md) supports drawing arcs between two 3D positions.
+- [TextLayer](/docs/layers/text-layer.md) adds a new prop `backgroundColor`. Picking works when the cursor is over an empty pixel in the text.
```
Comment (tgorkin) [inline docs/whats-new.md]: Has this been merged in?  I see this open PR - https://github.com/uber/deck.gl/pull/3903 @xintongxia

**Comment by Pessimistress** [comment]
@ajduberstein Can you add a blob for json/pydeck related updates?

**Comment by ajduberstein** [comment]
My summary:
```
#### pydeck 0.2.0
- Adds Jupyter widget multiselect support, allowing users to click multiple objects in a visualization and pass them to the Python backend
- Adds JupyterLab support
- Adds new error surfacing in the Jupyter UI
- Patches a bug around rendering GeoJSON passed as an object in the GeoJSONLayer (#3836)
- Patches a bug around string passing which affected `getTextAnchor` in the TextLayer (#3946)
- Supporting non-Mercator visualizations
```

**Comment by danmarshall** [comment]
Thank you for this @Pessimistress !! Are there any breaking changes in the API from 7.x or is 8.x just new features?

**Pull request merged**